modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/FilterSortAndSearchTheListOfAPIApplications.testcase

```diff
@@ -184,7 +184,7 @@ definition {
 		}
 
 		task ("Then the APIs are sorted by Status") {
-			APIBuilderUI.viewItemsInOrder(itemList = "test1,test2,test3");
+			APIBuilderUI.viewItemsInOrder(itemList = "test1");
```

Comment:
shouldn't we see test1,test3,test2?

Reply:
Actually, the results are not stable, sometimes test1,test2,test3, and sometimes test1,test3,test2, it's the test failed reasons. The dev said it has been sorted by status and it's not a bug, so I talked with Ashley, and we think the best way to solve the flaky is we only assert the first published application

Reply:
ok, well we're testing that it's being sorted by status, so why are not we checking the Status field? We should check whether it's PUBLISHED,PUBLISHED,UNPUBLISHED